packages/cozy-client/src/RealTimeQueries.jsx

```diff
@@ -26,16 +27,34 @@ const RealTimeQueries = ({ doctype }) => {
       )
     }
 
+    const handleCreated = data => {
+      let options = {}
+      if (doctype === 'io.cozy.files') {
+        options.computeCreatedDoc = ensureFilePath
+      }
+      dispatchCreate(client, doctype, data, options)
+    }
+
+    const handleUpdated = data => {
+      let options = {}
+      if (doctype === 'io.cozy.files') {
+        options.computeUpdatedDoc = ensureFilePath
+      }
+      dispatchUpdate(client, doctype, data, options)
+    }
+
+    const handleDeleted = data => {
+      let options = {}
+      if (doctype === 'io.cozy.files') {
+        options.computeDeletedDoc = ensureFilePath
```

Comment:
It's a detail, but I'm not sure it's worth naming differently each method with "delete", "create", "update"

Reply:
There's no advantage in doing this in-house now 👍